Package.swift

```diff
@@ -54,10 +54,10 @@ let cHeaderSearchPaths: [CSetting] = projectHeaderSearchPaths.map { .headerSearc
 let package = Package(
     name: "AppCenter",
     platforms: [
-        .iOS(.v11),
+        .iOS(.v12),
```

Comment:
no need to change this file, it's for older XCode. It's probably a good time to remove this file.

Reply:
done